--- xawdecode-1.9.2/font/Makefile.in.orig	2003-12-08 21:11:20.000000000 +0100
+++ xawdecode-1.9.2/font/Makefile.in	2004-08-22 13:33:43.025306560 +0200
@@ -1,7 +1,6 @@
-ROOT=
 prefix=/usr/X11R6
 #FONTDIR=$(ROOT)$(prefix)/lib/X11/fonts/misc
-FONTDIR=$(ROOT)/@x_libraries@/X11/fonts/misc
+FONTDIR=/usr/share/fonts/misc
 BDF=$(shell echo *.bdf)
 PCF=$(subst .bdf,.pcf,$(BDF))
 
@@ -9,15 +8,10 @@
 	$(MKFONTDIR)
 
 install: all
-	mkdir -p $(R)$(DESTDIR)$(FONTDIR)
+	mkdir -p $(DESTDIR)$(FONTDIR)
 	for file in $(PCF); do \
-		install -m 644 $$file $(R)$(DESTDIR)$(FONTDIR); \
+		install -m 644 $$file $(DESTDIR)$(FONTDIR); \
 	done
-	if test "$(ROOT)" = ""; then		\
-		(cd $(FONTDIR); $(MKFONTDIR));	\
-		xset fp rehash;			\
-		true;				\
-	fi
 
 uninstall:
 #	for file in $(PCF); do \
--- xawdecode-1.9.2/src/Makefile.am.orig	2004-06-18 20:12:33.000000000 +0200
+++ xawdecode-1.9.2/src/Makefile.am	2004-08-22 13:34:00.074118781 +0200
@@ -97,9 +97,6 @@
 xdtv_wizard_SOURCES = xdtv_wizard.c
 xdtv_wizard_LDADD = @X_LIBS@ @LIBXAW@ @LIBXMU@ @LIBXT@ @LIBXPM@ @LIBXEXT@ -lX11
 
-install-exec-local:
-	chmod u+s $(DESTDIR)$(prefix)/bin/xdtv_v4l-conf
-
 xdtv_scantv_LDADD = @LIBZVBI@ @LIBM@ @LIBPTHREAD@
 
 xdtv_scantv_SOURCES = xdtv_scantv.c grab-v4l.c grab-v4l2.c strtab.c colorspace.c frequencies.c
