--- xawdecode-1.9.2/configure.in.orig	2004-08-22 13:42:00.637222000 +0200
+++ xawdecode-1.9.2/configure.in	2004-08-22 13:57:52.488892135 +0200
@@ -549,9 +549,9 @@
 AC_SUBST(LIBM)
 
 LIBNET=""
-AC_CHECK_LIB(socket, socket, LIBNET="$LIBNET -lsocket",, $LIBNET)
-AC_CHECK_LIB(nsl, gethostbyname, LIBNET="$LIBNET -lnsl",, $LIBNET)
-AC_CHECK_LIB(resolv, herror, LIBNET="$LIBNET -lresolv",, $LIBNET)
+AC_CHECK_FUNC(socket,, [AC_CHECK_LIB(socket, socket, LIBNET="$LIBNET -lsocket",, $LIBNET)])
+AC_CHECK_FUNC(gethostbyname,, [AC_CHECK_LIB(nsl, gethostbyname, LIBNET="$LIBNET -lnsl",, $LIBNET)])
+AC_CHECK_FUNC(herror,, [AC_CHECK_LIB(resolv, herror, LIBNET="$LIBNET -lresolv",, $LIBNET)])
 AC_SUBST(LIBNET)
 
 AC_PATH_XTRA
