FFMPEG_VERSION is not defined in ffmpeg 20061204

--- xdtv-2.3.2/src/divx.c~	2006-01-21 00:05:36.000000000 +0200
+++ xdtv-2.3.2/src/divx.c	2006-12-14 10:33:58.552860560 +0200
@@ -1172,7 +1172,7 @@
     {div_t A_div, B_div;
     A_div=div(LIBAVCODEC_BUILD,65536);
     B_div=div(A_div.rem,256); 
-    sprintf(xawmesstmp, translate("divx_init: initializing %s version %s (build %i.%i.%i)\n"), codec_name, FFMPEG_VERSION, A_div.quot, B_div.quot, B_div.rem);}
+    sprintf(xawmesstmp, translate("divx_init: initializing %s version %s (build %i.%i.%i)\n"), codec_name, avcodec_version(), A_div.quot, B_div.quot, B_div.rem);}
 #endif
     strcat(xawmessage,xawmesstmp);
     switch(divx.codec) {
